solutions/2.1-bar_and_fill_between.py

```diff
@@ -1,8 +1,9 @@
+#import required libraries 
 import numpy as np
 import matplotlib.pyplot as plt
-np.random.seed(1)
+np.random.seed(1) # generates exact same random values as they are here inside .seed(value)
```

Comment:
Since I suppose the comment is mainly useful for absolute beginners, and this carries a bit of ambiguity (`.seed` does not generate anything, where is "here" (this line or this tutorial?)) an alternative would maybe be

     # Fix the random state to have the same numbers created whenever this code is being run.

Reply:
I suggest using what we use in Matplotlib examples:
```python
# Fixing random state for reproducibility
np.random.seed(19680801)
```